<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>221</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PiWriting</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Select an image to convert:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="select_file_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>112</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Select File</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>301</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Select a file that was mailed to PiWriting:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="select_file_btn_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>112</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Check Mail</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>621</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>To mail images/files to PiWriting, send an email with the file to cs3307group16testacct@outlook.com</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>The converted text is below</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>-10</y>
        <width>221</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PiWriting</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>311</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>310</y>
        <width>211</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Play Text to Speech</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>310</y>
        <width>211</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Save as Audio File (.wav)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>440</y>
        <width>211</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Save as Text File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>530</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>280</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Speaking now! (Turn up volume)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>440</y>
        <width>211</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>Save as Document (.docx)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>410</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Document has been saved!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>280</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Audio file has been saved!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>410</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Text file has been saved!</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>110</y>
        <width>311</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>70</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose a language to convert to:</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>90</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>French</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>90</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Spanish</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>370</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>340</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Enter your name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>390</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Enter your email address:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>420</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>300</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Email File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>450</y>
        <width>112</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Send Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>480</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Your email has been sent!</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>-10</y>
        <width>221</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>PiWriting</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
